examples/custom-renderer-codelab/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "renderer-sample-app",
+  "version": "1.0.0",
+  "description": "A sample app using Blockly",
```

Comment:
Update description to connect to the codelab somehow.

Reply:
yeah. this was on my todo list to change after maribeth's review on https://github.com/google/blockly-samples/pull/1601, just hadn't gotten to it yet.  good catch!